Examples/php/variables/runme.php

```diff
@@ -26,10 +25,10 @@
 	echo "pt	= ".pt_get(), point_print(pt_get()) , "\n";
 
 	/* Try to set the values of some global variables */
-$a = "42.14";
+    $a = "42.14";
 
 	ivar_set($a);
-echo "a = $a\n";
+    echo "a = $a\n";
```

Comment:
These two lines could do with reindenting (though more than you've reindented them) but that shouldn't be mixed in with functional changes.